Resources/views/Template/content-types/news-selection.html.twig

```diff
@@ -0,0 +1,12 @@
+<div id="{{ id|raw }}"
+     class="hide-in-sortmode"
+     data-property='{{ property|json_encode }}'
+     data-mapper-property="{{ property.name }}"
+     data-form="true"
+     data-type="news-selection"
+     data-type-instance-name="{{ id|raw }}"
+     data-aura-component="news-selection@examplenews"
+     data-aura-navigate-event="sulu.content.navigate"
+     data-aura-instance-name="{{ id|raw }}"
+     data-aura-url="/admin/api/news?flat=true&sortBy=title">
```

Comment:
in most cases you want also add a language code here: `&locale={{ languageCode }}`